guide.md

```diff
@@ -39,32 +39,30 @@ Illustration sources
+We designed this guide to help anyone navigate those complexities—regardless of where they live or work. We can’t imagine every possible use case, but we can help you build for them.
 
 ## What's in it?
 
-With Bitcoin being so new and different, a big focus of the guide lies on providing overviews, explanations and best practices for the most common questions and problems. These are the main chapters:
+The Bitcoin Design Guide is divided into the following sections:
 
-- [Getting started]({{ '/guide/getting-started/introduction/' | relative_url }}) is for top-level overviews, whether it's bitcoin technology, visual language, software, or design collaboration in an open-source world
-- [Onboarding]({{ '/guide/onboarding/introduction/' | relative_url }})  goes in-depth on the delicate process of introducing new users to bitcoin and how to use specific products and features
-- [Private key management]({{ '/guide/private-key-management/introduction/' | relative_url }}) provides extensive details and guidance on the many techniques for storing private keys
-- [Payments]({{ '/guide/payments/' | relative_url }}) deals with the complexities of sending and receiving bitcoin across various use cases
-- [Foundations]({{ '/guide/foundations/' | relative_url }}) highlights principles, best practices and other useful information for designing Bitcoin products
-- [Case studies]({{ '/guide/case-studies/' | relative_url }}) explores approaches to diverse uses cases with wireframes, designs and prototypes.
-- [Glossary]({{ '/guide/glossary/' | relative_url }}) explains common Bitcoin terminology, from Address to ZPUB.
+- [Getting started]({{ '/guide/getting-started/introduction/' | relative_url }}) provides top-level overviews of common visual language, software, and best practices for bitcoin design collaboration
+- [Onboarding]({{ '/guide/onboarding/introduction/' | relative_url }}) introduces new users to must know bitcoin terms, products, and features
+- [Private key management]({{ '/guide/private-key-management/introduction/' | relative_url }}) details the many techniques that can be used to store private keys
+- [Payments]({{ '/guide/payments/' | relative_url }}) covers the complexities encountered when sending and receiving bitcoin
+- [Foundations]({{ '/guide/foundations/' | relative_url }}) highlights principles and other fundamentals designing Bitcoin products
+- [Case studies]({{ '/guide/case-studies/' | relative_url }}) explores wire framing, designing, and prototyping for diverse use cases
```

Comment:
Turning the text around like this doesn't work. It's not an exploration of wire frames and other tools. It's an exploration of design solutions to certain use cases, using the means of wire framing and other tools.

Reply:
"lays out how to use wire frames and other tools to implement specific design-based use cases"

Reply:
Still not right. It's not how about which tools to use and how to use them. The section presents solutions to specific use cases. The solutions are not just described, but presented with design mock-ups, wire frames, prototypes, etc.

Reply:
So what do we do here then? Any concrete suggestion I can commit, as this is the only blocker to get this one merged.

Reply:
@GBKS Any thoughts on a next step here? Not 100% sure on what we’re trying to accomplish. If this is the only blocker, feel free to drop in any thoughts, or leave as is, and I can drop in feedback if necessary.

Reply:
Ping @GBKS

Reply:
If we have a better idea, happy to implement it, otherwise can revert as well, let's keep the PR's going this is really a minor blocker.

Reply:
Let's revert it. Sorry for the late reply.

Reply:
@GBKS done https://github.com/BitcoinDesign/Guide/pull/220/commits/a01e45a9349999f1e7b61ccacbdaa86edf6081f9